src/extension.ts

```diff
@@ -5,42 +5,12 @@ import * as vscode from 'vscode';
 const lurkTerminalName = 'Lurk REPL';
 
 let lurkTerminal: vscode.Terminal | null = null;
-let textQueue: string[];
-let onLoop = false;
 
-function queueLoop() {
-    if (lurkTerminal !== null) {
-        const text = textQueue.shift();
-        if (text) {
-            lurkTerminal.sendText(text);
-        }
-        if (textQueue.length > 0) {
-            // keep popping messages
-            onLoop = true;
-            setTimeout(queueLoop, 10);
-        } else {
-            onLoop = false;
-        }
-    } else {
-        onLoop = true;
-        setTimeout(queueLoop, 100);
-    }
-}
-
-function triggerLoop() {
-    if (onLoop) {
-        return;
-    }
-    queueLoop();
-}
-
-async function createLurkTerminal() {
+function getTerminal(): vscode.Terminal {
     // create lurk terminal if it doesn't exist
     if (lurkTerminal === null || lurkTerminal.exitStatus !== undefined) {
         const config: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration('lurkREPL');
         const lurkCommand = config.get("lurkRunCommand", "/home/user/.cargo/bin/lurk");
```

Comment:
Can we use `~/`, [`${userHome}`](https://code.visualstudio.com/docs/editor/variables-reference) or some other default for the home path here?